designer/server/src/test/scala/pl/touk/nussknacker/test/mock/TestCustomHttpServiceProviderFactory.scala

```diff
@@ -39,3 +50,115 @@ class SecondTestCustomHttpServiceProviderFactory extends CustomHttpServiceProvid
     Resource.pure(TestCustomHttpServiceProvider)
 
 }
+
+class TestTapirCustomHttpServiceProvider extends TapirCustomHttpServiceProvider {
+
+  import sttp.tapir._
+
+  import TestTapirCustomHttpServiceProvider._
+
+  private val tag = "Tapir based custom HTTP service"
+
+  private lazy val publicServerEndpointDefinition = new CustomHttpServiceServerEndpointDefinition {
+
+    type REQUEST  = Unit
+    type ERROR    = Unit
+    type RESPONSE = String
+
+    override def definition: Endpoint[Unit, Unit, Unit, String, Any] =
+      endpoint
+        .tag(tag)
+        .description("Public endpoint")
```

Comment:
I think that the name may be confusing after recent changes. The endpoint is no longer public, because all endpoints use authorization now.